src/components/DragonItem.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const dragonItem = (props) => {
+  /* eslint-disable */
+  const { name, type, img  } = props;
+  /* eslint-enable */
```

Comment:
- [ ] I suggest not disabling the linters. Linters ensure you use best practices in programming. Try resolving linter errors manually